### ISSUE TITLE
Missing Minecraft Download Server files

### ISSUE THREAD

**Issue by arek-kwapis** [Abstrat-Technologies/rustedanvil#15]
The following files are missing from libraries.minecraft.net

net/minecraft/launchwrapper/1.8/launchwrapper-1.8.jar
lzma/lzma/0.0.1/lzma-0.0.1.jar
com/paulscode/codecjorbis/20101023/codecjorbis-20101023.jar
com/paulscode/codecwav/20101023/codecwav-20101023.jar
com/paulscode/libraryjavasound/20101123/libraryjavasound-20101123.jar
com/paulscode/librarylwjglopenal/20100824/librarylwjglopenal-20100824.jar
com/paulscode/soundsystem/20120107/soundsystem-20120107.jar
argo/argo/2.25_fixed/argo-2.25_fixed.jar
org/lwjgl/lwjgl/lwjgl-platform/2.9.0/lwjgl-platform-2.9.0-natives-windows.jar
org/lwjgl/lwjgl/lwjgl-platform/2.9.0/lwjgl-platform-2.9.0-natives-osx.jar
org/lwjgl/lwjgl/lwjgl-platform/2.9.0/lwjgl-platform-2.9.0-natives-linux.jar
net/java/jinput/jinput-platform/2.0.5/jinput-platform-2.0.5-natives-windows.jar
net/java/jinput/jinput-platform/2.0.5/jinput-platform-2.0.5-natives-osx.jar
net/java/jinput/jinput-platform/2.0.5/jinput-platform-2.0.5-natives-linux.jar

**Comment by Unloaded123456** [comment]
Does launchwrapper-1.6 work? If it works, one of them will not be a problem.
Argo 2.25 is not a library meant for Minecraft, therefore, it should can be found in the author(s)' website. 
LWJGL 2.9.4 works well with 1.6.4, we do not need 2.9.0.

**Comment by arek-kwapis** [comment]
We don't need to edit what files are downloaded - all of them can be found and work just fine. It's just about automating the process

**Comment by Unloaded123456** [comment]
Automation really helps- manual installation is a nightmare. But if it can not be done, a text file that contains the links of the libraries of 1.6.4 would imitate automation.

**Comment by arek-kwapis** [comment]
Yeah, manual installation is a nightmare - it's the way I done it originally, and it's why I know all the files still exist 😋

We'll figure out ways of automation in Milestone 2 and seeing if it can be done or not. It's not important just now

**Comment by Unloaded123456** [comment]
Even they removed the files, the files would be archived elsewhere.

**Comment by Unloaded123456** [comment]
Perhaps it should be marked as "completed" in README.md
